src/lucky_cli/runner.cr

```diff
@@ -26,6 +27,9 @@ class LuckyCli::Runner
         task.call
       else
         TaskNotFoundErrorMessage.print(task_name)
+        if (@@exit_with_error_if_not_found)
```

Comment:
You could remove the parens and use `exit_with_error_if_not_found` instead of referencing the class variable directly